client/src/components/Page/ChooseYourRunPage.js

```diff
@@ -28,10 +29,15 @@ export default function ChooseYourRunPage() {
     fetchData();
   }, []);
 
+  function handleTypeIn(searchResults) {
+    setRuns(searchResults);
+  }
+
   return (
     <>
       <AppHeader title={"Choose Your Run"} />
       <ChooseRunContainer>
+        <Search handleSubmit={handleTypeIn} />
```

Comment:
`handleSubmit`, in this case would usually be called `onSubmit` if it is a function, that is called when something is submitted. That way, when implemented, the prop will look like: `onSubmit={handleSubmit}`, or in your case maybe `onSubmit={ () => setRuns(searchResults) }` because that is what you want to do on submit :)

Reply:
Thanks